app/navigation/Authenticator.js

```diff
@@ -29,42 +38,18 @@ const Authenticator = () => {
             component={Dashboard}
             options={{
               title: 'Econform',
-              headerStyle: {
-                backgroundColor: colors.darkGrey,
-              },
-              headerTintColor: '#fff',
-              headerTitleStyle: {
-                fontWeight: 'bold',
-              },
+              ...options,
```

Comment:
With the screen options you can pass this directly into the Stack.Navigator component using its screenOptions prop.

```
screenOptions={{
        ...options,
      }}
```